Tests/MSFT_xWindowsOptionalFeature.Tests.ps1

```diff
@@ -0,0 +1,311 @@
+<# 
+.summary
+    Test suite for MSFT_xWindowsOptionalFeature.psm1
+#>
+[CmdletBinding()]
+param()
+
+Import-Module $PSScriptRoot\..\DSCResources\MSFT_xWindowsOptionalFeature\MSFT_xWindowsOptionalFeature.psm1
```

Comment:
Consider using the [unit test template](https://github.com/PowerShell/DscResources/blob/master/Tests.Template/unit_template.ps1)